src/iop/diffuse.c

```diff
@@ -1514,98 +1515,85 @@ void gui_init(struct dt_iop_module_t *self)
   dt_bauhaus_slider_enable_soft_boundaries(g->radius, 0., 2048.);
   dt_bauhaus_slider_set_format(g->radius, "%.0f px");
   gtk_widget_set_tooltip_text(
-      g->radius, _("width of the diffusion around the center radius.\n"
+      g->radius, _("width of the diffusion around the central radius in px.\n"
                    "high values diffuse on a large band of radii.\n"
-                   "low values diffuse closer to the center radius.\n"
-                   "if you plan on deblurring, the radius should be around the width of your lens blur."));
-
-  gtk_box_pack_start(GTK_BOX(self->widget), dt_ui_section_label_new(_("diffusion speed")), FALSE, FALSE, 0);
+                   "low values diffuse closer to the central radius.\n"
+                   "if you plan on deblurring, \n"
+                   "the radius should be around the width of your lens blur."));
+
+  GtkWidget *label_speed = dt_ui_section_label_new(_("speed (sharpen ↔ diffuse)"));
+  gtk_box_pack_start(GTK_BOX(self->widget), label_speed, FALSE, FALSE, 0);
+  gtk_widget_set_tooltip_text(label_speed, _("negative values sharpen, \n"
+                                          "positive values diffuse and blur, \n"
+                                          "zero does nothing."));
 
   g->first = dt_bauhaus_slider_from_params(self, "first");
   dt_bauhaus_slider_set_factor(g->first, 100.0f);
   dt_bauhaus_slider_set_digits(g->first, 4);
   dt_bauhaus_slider_set_format(g->first, "%+.2f %%");
-  gtk_widget_set_tooltip_text(g->first, _("smoothing or sharpening of smooth details (gradients).\n"
-                                          "positive values diffuse and blur.\n"
-                                          "negative values sharpen.\n"
-                                          "zero does nothing."));
+  gtk_widget_set_tooltip_text(g->first, _("speed of low-frequency wavelet layers\n"
+                  "in the direction of 1st order anisotropy (set below)."));
```

Comment:
I understand you're trying to remove repetition here but honestly it might be better to repeat than to make people move around to view another tooltip.

Reply:
It's not just about removing repetition, but making it easier to understand the tooltips.
If I want to know, what's the difference between the 4 sliders, I have to filter out everything which is identical, in order to see the differences.
Currently the start and the end of the tooltips are identical for all 4 sliders, but there are small differences inbetween.

As an alternative I now start with the differences, and after an empty line add the repetition.